### ISSUE TITLE
[A11y][ScreenReader]NVDA is announcing as the Upload package heading twice

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9171]
Original: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1554248

Repro Steps:
Hit https://www.nuget.org and press Enter key.
Navigate to "Sign in" control and hit Enter key.
Navigate to "Sign in with Microsoft" control and hit Enter key.
Select “Upload Package” from the Profile Dropdown and hit Enter key.
Verify that whether the Upload package heading is announcing as two times or not.

Actual Result:
NVDA is announcing as the Upload package heading as twice when the NVDA focus receives.

Expected result:
NVDA should be announce the Upload package as once when the NVDA focus should receive.

User Impact:
Screen Reader Users may be get confused when the screen reader should announce twice the heading when the focus receives.

Notes:
Unsure why this is happening. It is likely taht the focused element has content, and that a child element also has the content, and both are being read out. I believe this is a way to mark some information as ignore to screenreaders. Assuming this is the case, marking the child information as do not read should fix this.

**Comment by camigthompson** [comment]
Unable to reproduce. Replied to ADO bug asking for more information.

**Comment by camigthompson** [comment]
Still awaiting a response on ADO bug.

**Comment by camigthompson** [comment]
ADO bug closed.
